Tests/KituraTests/TestCustomCoders.swift

```diff
@@ -152,8 +154,13 @@ class TestCustomCoders: KituraTest {
         
         performServerTest(customRouter) { expectation in
             self.performRequest("get", path: "/rawget", callback: { response in
-                if let response = response, let responseString = try? response.readString() {
-                    XCTAssertEqual(responseString, "&date=2018-02-21T09:47:36%2B0000")
+                if let response = response,
+                   let responseString = try? response.readString(),
+                   let unwrappedString = responseString
+                {
+                    let responseDate = self.dateFormatter.date(from: String((unwrappedString.dropFirst(6))))
```

Comment:
Could you add a comment explaining why we are dropping the first 6 characters? (presumably, `&date=`)

Reply:
Have added a comment for why we drop the first 6 characters.